InnerEye/Scripts/move_model.py

```diff
@@ -0,0 +1,127 @@
+MODEL_PATH = "MODEL"
+ENVIRONMENT_PATH = "ENVIRONMENT"
+MODEL_JSON = "model.json"
+
+
+@dataclass
+class MoveModelConfig(GenericConfig):
+    model_id: str = param.String(default="model_id",
+                                 doc="AzureML model_id")
+    path: str = param.String(default="path",
+                             doc="Path to import or export model")
+    action: str = param.String(default="action",
+                               doc="Import or export model from workspace. E.g. import or export")
+
+
+def get_paths(path: Path, model_id: str) -> Tuple[Path, Path]:
+    """
+    Gets paths and creates folders if necessary
+    :param path: Base path
+    :param model_id: The model ID
+    :return: model_path, environment_path
+    """
+    model_id_path = Path(path) / model_id.replace(":", "_")
+    model_id_path.mkdir(parents=True, exist_ok=True)
+    model_path = model_id_path / MODEL_PATH
+    model_path.mkdir(parents=True, exist_ok=True)
+    env_path = model_id_path / ENVIRONMENT_PATH
+    env_path.mkdir(parents=True, exist_ok=True)
+    return model_path, env_path
+
+
+def download_model(ws: Workspace, config: MoveModelConfig) -> Model:
+    """
+    Downloads an InnerEye model from an AzureML workspace
+    :param ws: The AzureML workspace
+    :param config: move config
+    :return: the exported Model
+    """
+    model = Model(ws, id=config.model_id)
+    model_path, environment_path = get_paths(Path(config.path), config.model_id)
+    with open(model_path / MODEL_JSON, 'w') as f:
+        json.dump(model.serialize(), f)
+    model.download(target_dir=str(model_path))
+    env_name = model.tags.get(PYTHON_ENVIRONMENT_NAME)
+    environment = ws.environments.get(env_name)
+    environment.save_to_directory(str(environment_path), overwrite=True)
+    return model
+
+
+def upload_model(ws: Workspace, config: MoveModelConfig) -> Model:
+    """
+    Uploads an InnerEye model from an AzureML workspace
+    :param ws: The AzureML workspace
+    :param config: move config
+    :return: imported Model
+    """
+    model_path, environment_path = get_paths(Path(config.path), config.model_id)
+    with open(model_path / MODEL_JSON, 'r') as f:
+        model_dict = json.load(f)
+
+    new_model = Model.register(ws, model_path=str(model_path / "final_model"), model_name=model_dict['name'],
+                               tags=model_dict['tags'], properties=model_dict['properties'],
+                               description=model_dict['description'])
+    env = Environment.load_from_directory(str(environment_path))
+    env.register(workspace=ws)
+    print(f"Environment {env.name} registered")
+    return new_model
+
+
+def get_workspace(config: MoveModelConfig) -> Workspace:
+    return Workspace.get(name=config.workspace_name, subscription_id=config.subscription_id,
+                         resource_group=config.resource_group)
+
+
+def main() -> None:
+    config = MoveModelConfig.parse_args()
```

Comment:
There are no consistency checks for the parsed args, all of the fields are required but this is not checked at all.